site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+An introduction to Sphinx and Read the Docs for Technical Writers
+=================================================================
+
+By: Eric Holscher
+
+Bio: Eric Holscher is the co-founder of Read the Docs, and the Write the Docs conference.
```

Comment:
think this needs to drop `,` or add a qualifier: `organizer of the Write the Docs conference`